apax/train/run.py

```diff
@@ -194,15 +38,26 @@ def setup_logging(log_file, log_level):
     logging.basicConfig(filename=log_file, level=log_levels[log_level])
 
 
+def initialize_directories(model_version_path: Path) -> None:
+    log.info("Initializing directories")
+    os.makedirs(model_version_path, exist_ok=True)
```

Comment:
if the `model_version_path` is truely a `Path` why not use `Path.mkdir(exists_ok=True)`

Reply:
good point. The function is pretty trivial by now. In the past it used to do much more. I will just replace it with an inline log and Path.mkdir call